kernel/arch/i386/paging_asm.s

```diff
@@ -0,0 +1,14 @@
+.section .text
+.align 4
+
+.global px_invalidate_page
+px_invalidate_page:
+#if TARGET_MACHINE >= TARGET_MACHINE_I486
```

Comment:
What is the purpose of this section?

Reply:
To clear the TLB

Reply:
As per what we discussed, the invalidate page instruction does not exist on anything lower than i486, so this is necessary. Closing.